Description: Allow EnvIsSet tests to be performed on InfoStore variables
 A patch from upstream for a minor oversight, not yet released.
Origin: https://github.com/fvwmorg/fvwm/commit/d950dabf4d815fb59babe2b40200f52b248a04c8
Author: tsundoku <me@tsundoku.ne.jp>
Last-Update: 2019-02-19
--- a/fvwm/conditional.c
+++ b/fvwm/conditional.c
@@ -2101,8 +2101,15 @@
 			flags_ptr = GetNextSimpleOption(flags_ptr, &var_name);
 			if (var_name)
 			{
-				const char *value = getenv(var_name);
-
+				const char *value;
+				if ( (strlen(var_name) > 10) && (memcmp(var_name,"infostore.",10) == 0)  )
+				{
+					value = get_metainfo_value(var_name + 10);
+				}
+				else
+				{
+					value = getenv(var_name);
+				}
 				match = (value != NULL) ? 1 : 0;
 			}
 			else
